site/src/pages/TemplatePage/TemplatePageView.tsx

```diff
@@ -56,7 +56,7 @@ export const TemplatePageView: FC<React.PropsWithChildren<TemplatePageViewProps>
     <Margins>
       <PageHeader
         actions={
-          <Stack direction="row" spacing={1}>
+          <>
```

Comment:
Why this change?

Reply:
The PageHeader is already wrapping the actions inside of a Stack.